app/js/controllers/admin/team.js

```diff
@@ -0,0 +1,78 @@
+const AdminTeamCtrl = function ($scope, $state, $mdDialog, $mdToast, AdminDataService, ClientDataService, $sce) {
+
+    $scope.team = getTeam();
+    $scope.selected = [];
+    //utlility methods
+    $scope.trustAsHtml = (template) => {
+        return $sce.trustAsHtml(template);
+    };
+
+    $scope.fetchTeam = () => {
+        getTeam();
```

Comment:
Again here, I feel the project implementation was better for fetching the projects and adding them to the scope.